x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -63,9 +62,9 @@ const AppRoot: React.FunctionComponent<RouterProps> = React.memo(
         <I18nProvider>
           <KibanaContextProvider services={{ http, notifications, application, data }}>
             <EuiThemeProvider darkMode={isDarkMode}>
-              <BrowserRouter basename={basename}>
+              <Router history={history}>
```

Comment:
why did we stop using `<BrowserRouter>` ?

Reply:
It was a change request pushed down from the Kibana Platform - see #56705

Reply:
from the ticket, looks like they still use `<BrowserRouter>` in the after:

```
core.application.register({
  id: 'myApp',
  mount({ element, history }) {
    ReactDOM.render(
      <BrowserRouter history={history}>
        <App />
      </BrowserRouter>,
      element
    );
    return () => ReactDOM.unmountComponentAtNode(element);
  }
});
```

Reply:
@kevinlog Thats a mistake. BrowserRouter does not accept the `history` prop 😃

Reply:
got it, sounds good